{{cookiecutter.project_name}}/.github/next_steps/05_linting.md

```diff
@@ -2,4 +2,13 @@
 title: 'Next step: Linting'
 ---
 
-Linting instructions
+Your repository has a [workflow]({{ cookiecutter.repository }}/blob/main/.github/workflows/lint.yml) which [lints](https://en.wikipedia.org/wiki/Lint_(software)) your code in every push and when creating a pull request.
+
+To make sure you have a proper linter configuration you need to follow the steps below.
```

Comment:
is it configuration? 1. is to make linter happy and point 2 is about keeping the linter happy

Reply:
updated the text